Vizzuality/i2i
# active layers indicator deleted

⚠️ Before submitting the PR, make sure:

- You are including only one feature
- Commits messages are clear
- You updated the changelog
- You asked someone to review it

---

Description of the PR

## Testing instructions

Go to map and select a layer in the sidebar, the layer indicator should not appear.

http://localhost:3000/data-portal/ZMB/fsp-maps?lat=-13.2399454992863&layers=%5B%7B%22id%22%3A%222019071520%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%2C%22visualizationType%22%3A%22normal%22%7D%2C%7B%22id%22%3A%222019071521%22%2C%22opacity%22%3A1%2C%22visibility%22%3Atrue%2C%22visualizationType%22%3A%22normal%22%7D%5D&lng=27.685546875000004&zoom=3

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/169184957
